sopel_modules/rat_board.py

```diff
@@ -856,8 +856,8 @@ def func_clear(bot, trigger, rescue, markingForDeletion=False, *firstlimpet):
     if not markingForDeletion and (not rescue.platform or rescue.platform == 'unknown'):
         bot.say('The case platform is unknown. Please set it with the corresponding command and try again.')
         return
-    url = "https://fuelrats.com/paperwork/{rescue.id}/edit".format(
-        rescue=rescue, apiurl=str(bot.config.ratbot.apiurl).strip('/'))
+
+    url = "https://fuelrats.com/paperwork/{rescue.id}/edit".format(rescue=rescue)
```

Comment:
did you intentionally remove the access to a configuration object? this change makes a configuration setting do nothing if i understand the delta correctly.

Reply:
... 🤔 im confused on if that config value was even /used/ by that delta

Reply:
it was not. It was legacy code from v1 days

Reply:
rip. well imma just mark this comment as resolved then.